dev/tests/api-functional/testsuite/Magento/Customer/Api/CustomerRepositoryTest.php

```diff
@@ -31,16 +31,11 @@
  */
 class CustomerRepositoryTest extends WebapiAbstract
 {
-    const SERVICE_VERSION = 'V1';
-    const SERVICE_NAME = 'customerCustomerRepositoryV1';
-    const RESOURCE_PATH = '/V1/customers';
-    const RESOURCE_PATH_CUSTOMER_TOKEN = "/V1/integration/customer/token";
+    private const SERVICE_VERSION = 'V1';
+    private const SERVICE_NAME = 'customerCustomerRepositoryV1';
+    private const RESOURCE_PATH = '/V1/customers';
```

Comment:
Not sure that we're allowed to change the `const` visibility. As result, looks like other tests rely on these constants (please check the failing WebAPI tests).